### DISCUSSION TITLE
Show changelog on auto update

### DISCUSSION THREAD

**Discussion by ghost** [grumpycoders/pcsx-redux#1082] [Ideas]
When both "Auto Update" is enabled and there is a new update do more than just notify the PCSX-Redux user that there exists a new update. Also show him a changelog, what's new? What was added, what was removed, what was changed and etc.

**Comment by wheremyfoodat** [comment]
A little irrelevant to the request at hand (since it's already been discussed in a previous discussion) but could you try using the issues tab for feature requests rather than the discussion tab? There's a variety of reasons why discussions don't provide the necessary infrastructure for managing feature requests.

- Discussions don't let you assign tasks to specific individuals, issues do.
- Discussions are very hard to track as you can't close them. You can only lock the discussion which is more of a hindrance than a utility. 
- Certain GH clients don't support discussions (eg I use FastHub on Android which doesn't)
- On clients that do, people forget the tab exists
- Filtering issues and distinguishing between open and closed ones is a lot easier than discussions, + the issues tab enumerates how many are open at a time.

Of course the relevant precautions need to be taken in order to avoid flooding the issues tab as well.

**Reply by ghost** [reply]
> A little irrelevant to the request at hand (since it's already been discussed in a previous discussion) but could you try using the issues tab for feature requests rather than the discussion tab? There's a variety of reasons why discussions don't provide the necessary infrastructure for managing feature requests.
> 
> * Discussions don't let you assign tasks to specific individuals, issues do.
> * Discussions are very hard to track as you can't close them. You can only lock the discussion which is more of a hindrance than a utility.
> * Certain GH clients don't support discussions (eg I use FastHub on Android which doesn't)
> * On clients that do, people forget the tab exists
> * Filtering issues and distinguishing between open and closed ones is a lot easier than discussions, + the issues tab enumerates how many are open at a time.
> 
> Of course the relevant precautions need to be taken in order to avoid flooding the issues tab as well.

I didn't know about these facts that you mentioned above. I thought that every new post that neither reports nor describes an issue, but proposes, suggests and talks about a new idea logically should be posted in the Discussions under the "Ideas" category.

Strange that there was no (1) in this google chrome tab so I didn't return here to read this reply earlier and I kept doing what you did not recommend me to do!

Perhaps I must check again all open tabs to see if there is any new reply from you and others too even if I don't see (1) in the open tab!

**Reply by lorn10** [reply]
@wheremyfoodat Just FYI it is in the meantime possible to convert an issue into a discussion and this might be true also for the other way around. So to convert a discussion into an issue and then close it when it is resolved. :thinking: 

It looks that quite several of the discussions started by @buruto are in fact _feature requests_ or _suggestions for improvements_. 

See also: 
`
https://github.com/lxqt/lxqt/issues/2468`
